smarts/core/agent_interface.py

```diff
@@ -135,8 +135,8 @@ class AgentType(IntEnum):
     """Agent performs trajectory tracking using model predictive control."""
     TrajectoryInterpolator = 11
     """Agent performs linear trajectory interpolation."""
-    Imitation = 12
-    """Agent sees neighbor vehicles and performs actions based on imitation-learned model (acceleration, angular_velocity)."""
+    Direct = 12
+    """Agent sees neighbor vehicles and performs actions based on direct updates to (acceleration, angular_velocity)."""
```

Comment:
Yeah, I struggled with this name.  I think that "Imitation" is definitely the _wrong_ name for the action space (b/c it's too specific and assumes too much about the use case).  But I am also not sure that "Direct" is the best alternative.

But I disagree with what you wrote too:  neither acceleration nor angular_velocity are _forces_ -- and they aren't _applied_ to the vehicle chassis.  Indeed, this point is partly why I ended naming it "direct":  because there is no physical vehicle model that mediates between the "action" specified by the user and the realization of that.  (If the user asks for a specific acceleration and angular_velocity on a timestep, SMARTS complies without question.)

But this is the point were were discussing in our last Tuesday meeting (albeit w.r.t. a different action space).  I also think there _should_ be some sort of ("gentle") mediation, even with `BoxChassis`, to prevent unrealistic values from being applied.  And _IF_ we are going to add such mediation, _THEN_ I would definitely want to change the name of this action space to be something besides "Direct".

But I don't have a good candidate name yet for that eventuality...  I'm open to suggestions!